lib/main.coffee

```diff
@@ -0,0 +1,25 @@
+
+module.exports =
+  provideLinter: ->
+    jshintPath = require('path').join(__dirname, '..', 'node_modules', '.bin', 'jshint')
```

Comment:
`execPath` is a great option, but it's not what I was thinking. I'm going to merge this, make the update to `eslint` and drop another PR with my thoughts it will be easier.